fdbrpc/IRateControl.h

```diff
@@ -37,8 +37,7 @@ class IRateControl {
 // An IRateControl implemenation that allows at most hands out at most windowLimit units of 'credit' in windowSeconds seconds
 class SpeedLimit final : public IRateControl, ReferenceCounted<SpeedLimit> {
 public:
-	SpeedLimit(int windowLimit, int windowSeconds) : m_limit(windowLimit), m_seconds(windowSeconds), m_last_update(0), m_budget(0) {
-		m_budget_max = m_limit * m_seconds;
+	SpeedLimit(int windowLimit, double windowSeconds) : m_limit(windowLimit), m_seconds(windowSeconds), m_last_update(0), m_budget(0) {
 		m_last_update = timer();
```

Comment:
I think I wrote this before I knew that `timer()` breaks determinism, this should be `now()`.

Reply:
There's also the new `g_network->timer()` if you need determinism and time advancing during the same task